README.md

```diff
@@ -45,6 +45,21 @@ The goals of LangOCR are to do the following.
   operates.
 
 ## Installation
+First, add the bincrafters remote for Conan:
```

Comment:
Please add a hyperlink to Conan documentation so people who are not enlightened can get and idea what Conan is.